class.jetpack-admin.php

```diff
@@ -95,6 +95,10 @@ function get_modules() {
 				}
 				$module_array['long_description']  = ob_get_clean();
 
+				ob_start();
+				do_action( 'jetpack_search_terms_' . $module );
```

Comment:
May be best to do in a separate PR as to not take away the focus of Search Terms here